.dotnet/src/Generated/Models/Assistant.Serialization.cs

```diff
@@ -115,9 +139,12 @@ protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWrit
                     writer.WriteNull("temperature"u8);
                 }
             }
-            writer.WritePropertyName("object"u8);
+            if (_additionalBinaryDataProperties?.ContainsKey("object") != true)
+            {
+                writer.WritePropertyName("object"u8);
```

Comment:
This doesn't look correct. The write statement should be in the condition right?